scripts/build.js

```diff
@@ -164,6 +164,15 @@ manifest.web_accessible_resources?.forEach((entry) => {
   });
 });
 
+// declarative net request
+if (manifest.declarative_net_request?.rule_resources) {
+  manifest.declarative_net_request.rule_resources.forEach(({ path }) => {
+    const dir = dirname(path);
+    shelljs.mkdir('-p', resolve(options.outDir, dir));
+    shelljs.cp(resolve(options.srcDir, path), resolve(options.outDir, dir));
```

Comment:
based on a target (chrome/safari) we can copy only `dnr` or `dnr-safari` lists to reduce the extension size

Reply:
It works like this out of the box, as it uses costuimzed manifest per platform :)